kevm-pyk/.gitignore

```diff
@@ -1,3 +1,11 @@
 /dist/
 __pycache__/
 .coverage
+src/tests/unit/test-data/foundry-list/cache/
+src/tests/unit/test-data/foundry-list/lib/
+src/tests/unit/test-data/foundry-list/out/
+!src/tests/unit/test-data/foundry-list/out/apr_proofs/
+!src/tests/unit/test-data/foundry-list/out/kompiled/
+!src/tests/unit/test-data/foundry-list/out/kompiled-llvm/
+!src/tests/unit/test-data/foundry-list/out/tmp/
+!src/tests/unit/test-data/foundry-list/out/digest
```

Comment:
```suggestion
```

Reply:
Should we remove all those ignored folders ? We probably don't need to commit cache for instance, as well as {kompiled, kompiled-llvm, tmp} as they have shown to be unnecessary

Reply:
Right, but they shouldn't be there in the local tree either.

Reply:
Totally makes sense! Though, how do we prevent other people from committing them  once they launch the unit test ?

Reply:
My point is, unit tests should not generate these files. Whatever you need to run `foundry-list` has to be provided as a minimal set of files as test data.

Reply:
Of course, Thanks!